<?xml version="1.0" ?>
<launch>
  <arg name="log_dir" default="" />
  <arg name="sim" default="true" />
  <arg name="robot" default="pr2" />

  <node name="object_search_manager" pkg="race_object_search" type="object_search_manager" output="screen" respawn="true">
    <!-- global or world frame id -->
    <param name="world_frame_id" type="string" value="/map" />
    <!-- time needed for sensor data integration and object recognition -->
    <param name="acquisition_time"   type="double" value="5.0" />
    <!-- additional integration timeout to work around lag in simulated sensors -->
    <param     if="$(arg sim)" name="additional_acquisition_time" type="double" value="5.0" />
    <param unless="$(arg sim)" name="additional_acquisition_time" type="double" value="0.0" />
    <!-- name of the robot agent to use -->
    <param name="robot" type="string" value="$(arg robot)" />
    <!-- directory to place log outout -->
    <param name="log_dir" type="string" value="$(arg log_dir)" />
    <!-- set planning mode. available: search, iw, id -->
    <param name="planning_mode" type="string" value="search" />
    <!-- maximum depth to allow branching -->
    <param name="depth_limit" type="int" value="2" />
    <!-- portion of search progress upon which a plan is regarded as complete -->
    <param name="relative_lookahead" type="double" value="1.0" />
    <!-- relative utility limit for branching -->
    <param name="max_rel_branch_cost" type="double" value="1.3" />
    <!-- hard planning timeout -->
    <param name="planning_timeout" type="double" value="10" />
    <!-- number of local and global sensor pose samples -->
    <param name="local_sample_size" type="int" value="0" />
    <param name="global_sample_size" type="int" value="200" />
    <!-- toggle domination pruning heuristic on/off -->
    <param name="use_domination" type="boolean" value="false" />
  </node>
</launch>
